composable_prepare_nic_configs.yml

```diff
@@ -230,10 +234,10 @@
         when: osp_release|int < 17
 
       - block:
-          - name: fail if ceph_machine_type or ceph_ifaces are not defined
+          - name: fail if ceph_machine_type
```

Comment:
name not giving any meaning

Reply:
I am thinking of moving this type of check at the beginning of the installation, instead of doing it halfway. thoughts?

Reply:
Done